### PR TITLE
New article: Create a custom intelligence dashboard

### PR DESCRIPTION
**Pull request by DonNateR**
Fixes issue #2168

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a5385159dd/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c011330b60/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-24d6569752/how-to/)

File: content/rackspace-intelligence/create-a-custom-intelligence-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+type: article
+created_date: '2017-07-25'
+created_by: Nate Archer
+last_modified_date: '2017-07-25'
+last_modified_by: Nate Archer
+product: Rackspace Intelligence
+product_url: rackspace-intelligence
+---
+
+Rackspace Intelligence custom dashboards enable user's to create to get a better view of their infrastructure through customized graphs and alerts.
```
Comment (itzg) [inline content/rackspace-intelligence/create-a-custom-intelligence-dashboard.md]: Grammar: user's -> users

File: content/rackspace-intelligence/create-a-custom-intelligence-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Custom dashboards use the same checks and metrics that are used by Rackspace Monitoring. Create multiple dashboards to address different needs. If you to understand how servers in an application are performing, create a dashboard with graphs and alerting information for the servers in that application. Alternatively, create a dashboard to monitor specific metrics for database and production servers.
```
Comment (itzg) [inline content/rackspace-intelligence/create-a-custom-intelligence-dashboard.md]: perhaps change "that are used by" to "that are already available in"

Comment (itzg) [inline content/rackspace-intelligence/create-a-custom-intelligence-dashboard.md]: 3rd sentence: "If you to" -> "If you need to"

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-53f9803bd7/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Please let me know if you have any questions. Thanks!

[Create a custom intelligence dashboard_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1178394/Create.a.custom.intelligence.dashboard_editsKH.pdf)
[Create a custom intelligence dashboard_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1178395/Create.a.custom.intelligence.dashboard_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-77e970446d/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-79a70114c3/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b63ad2d335/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0a2d0ce37d/how-to/)

File: content/rackspace-intelligence/create-custom-intelligence-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+This following article provides instructions on how to create, access, and edit custom dashboards.
+
+### Create a custom dashboard
+
+1. Log into the Cloud Control Panel.
+2. In the top navigation bar, select **Servers** > **Rackspace Intelligence**.
+3. At the top of the Rackspace Intelligence page, click **Dashboard**.
+4. In the dashboard page, click **Custom**.
+5. On the Create Your Custom Dashboard page, provide a descriptive name for your dashboard and a brief description of it.
+6. In the **Start From** menu, chose whether your dashboard should include any monitoring widgets by default.
```
Comment (stephamon) [inline content/rackspace-intelligence/create-custom-intelligence-dashboards.md]: "chose" should be "choose"

File: content/rackspace-intelligence/create-custom-intelligence-dashboards.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+#### Add performance graphs
+
+Performance graphs show the relationships among various check types and metrics for specific entities. For more information about check types, see [Check types reference](https://developer.rackspace.com/docs/rackspace-monitoring/v1/tech-ref-info/check-type-reference/). For more information about metrics, see the Metrics section of the [Rackspace Monitoring API reference](https://developer.rackspace.com/docs/rackspace-monitoring/v1/api-reference/metrics-operations/).
+
+Use the following process to add a performance graph to your custom dashboard:
+
+1. Under **Categories**, click **Performance Graphs**.
+
+   Check types and metrics are based on the same check types and metrics in Rackspace Monitoring. Each check type contains its own set of metrics
```
Comment (stephamon) [inline content/rackspace-intelligence/create-custom-intelligence-dashboards.md]: Missing period at the end of the sentence.

File: content/rackspace-intelligence/create-custom-intelligence-dashboards.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+2. Click one of the following tabs:
+
+   - **Explore a Metric**: Creates a performance graph that focuses on one check type and one metric. The graph compares that metric across multiple entities (or devices such as servers or databases).
+   - **Explore an Entity**: Creates a performance graph with one entity and one check type. The graph examines multiple metrics for the selected entity.
+
+3. Select the values for the graph or graphs from the list of options::
```
Comment (stephamon) [inline content/rackspace-intelligence/create-custom-intelligence-dashboards.md]: extra colon

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4b273803d6/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**